src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let results = days * 40;
```

Comment:
Consider using a constant for the value `40` to improve code readability and maintainability.